packages/vue/src/components/organisms/SfFooter/_internal/SfFooterColumn.spec.js

```diff
@@ -0,0 +1,9 @@
+import "./matchMedia.mock";
```

Comment:
Why is this needed? It causes 'Cannot find module' error

Reply:
I didn't add this there

Reply:
Great :)